<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDesktopWallpaperWidget</class>
 <widget class="QWidget" name="MusicDesktopWallpaperWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌面壁纸设置</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>465</width>
     <height>360</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>桌面壁纸设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>347</y>
     <width>445</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>44</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{border:1px solid #FFF;}</string>
   </property>
   <property name="title">
    <string>设置壁纸来源方式</string>
   </property>
   <widget class="QRadioButton" name="localRadioButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启用本机图片做为壁纸</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="netRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启用网络图片做为壁纸</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="playRadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>43</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启用正在播放歌曲背景图片做为壁纸</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="pictureType">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>154</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlLineEdit">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>127</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>129</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择壁纸路径：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeH">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>184</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>157</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片位置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>187</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置时间间隔：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeM">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>184</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="timeS">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>184</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pictureFunc">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>214</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pictureEffect">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>244</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="openWithstart">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>284</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开机随系统启动</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="recoveryWallpaper">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>284</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭软件时还原初始壁纸</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>217</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>壁纸更换方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>247</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>壁纸切换效果：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>127</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>314</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>465</width>
     <height>335</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>line_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>pictureType</zorder>
  <zorder>urlLineEdit</zorder>
  <zorder>label</zorder>
  <zorder>timeH</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>timeM</zorder>
  <zorder>timeS</zorder>
  <zorder>pictureFunc</zorder>
  <zorder>pictureEffect</zorder>
  <zorder>openWithstart</zorder>
  <zorder>recoveryWallpaper</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>confirmButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>viewButton</zorder>
  <zorder>stopButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
